network/src/commonMain/kotlin/com/wire/kalium/network/session/SessionManager.kt

```diff
@@ -70,6 +71,8 @@ fun HttpClientConfig<*>.installAuth(sessionManager: SessionManager, accessTokenA
                         }
                         null
                     }
+                }.also {
+                    kaliumLogger.d("AUTH TOKEN REFRESH:{response_code: \"${response.status.value}\"}")
```

Comment:
@MohamadJaara @yamilmedina   that's not json :)

"AUTH TOKEN REFRESH: { \"response_code\": ${response.status.value} }"